tracer/build/crank/Security.Samples.AspNetCoreSimpleController.yml

```diff
@@ -59,12 +58,28 @@ scenarios:
         COR_ENABLE_PROFILING: 1
         CORECLR_ENABLE_PROFILING: 1
         DD_APPSEC_ENABLED: true
-        DD_APPSEC_BLOCKING_ENABLED: false
         DD_APPSEC_KEEP_TRACES: false
     load:
       job: bombardier
       variables:
         warmup: 30
         duration: 240
         serverPort: 5000
-        path: /hello?[$slice]=arg
\ No newline at end of file
+        path: /hello?[$slice]=arg
+
+  appsec_attack_blocking:
+    application:
+      job: server
+      environmentVariables:
+        COR_ENABLE_PROFILING: 1
+        CORECLR_ENABLE_PROFILING: 1
+        DD_APPSEC_ENABLED: true
+        DD_APPSEC_KEEP_TRACES: false
+        DD_APPSEC_RULES: ../../../../Datadog.Trace.Security.IntegrationTests/ruleset.3.0.json
```

Comment:
You don't need to set `DD_APPSEC_BLOCKING_ENABLED`? It's in the rules?

Reply:
Yes, blocking is no controlled by the WAF, and the rules that it runs. There doesn't seem to need to be safety mechanism as it's off by default and customers need to do work to opt in.